server/src/main.rs

```diff
@@ -1,17 +1,55 @@
+use clap::{crate_version, App, SubCommand};
 use stackable_operator::crd::CustomResourceExt;
-use stackable_operator::{client, error};
+use stackable_operator::{cli, client};
 use stackable_spark_crd::SparkCluster;
 use stackable_spark_crd::{Restart, Start, Stop};
 use tracing::{error, info};
 
 #[tokio::main]
-async fn main() -> Result<(), error::Error> {
+async fn main() -> anyhow::Result<()> {
     stackable_operator::logging::initialize_logging("SPARK_OPERATOR_LOG");
 
     info!("Starting Stackable Operator for Apache Spark");
 
+    // Handle CLI arguments
+    let matches = App::new("Spark Operator")
+        .author("Stackable GmbH - info@stackable.de")
```

Comment:
Might be worthwhile pulling the author from a commons crate as constant at some point?

Reply:
Yeah, at some point, yes.